### ISSUE TITLE
Simplify VTT translations management

### ISSUE THREAD

**Issue by remibetin** [w3c/wai-translations#117]
### TL:DR
- We add a YAML file to list all the VTTs' types and languages in the repo
- In a resource containing a video, we use a different video-player include, to automatically pull the available VTTs, and associate them with the video player.
- When a translator translates a VTT file, they only need to edit the YAML file. Every version of the page will then be automatically updated to provide the new subtitles/descriptions.
- When a user visits the resource, VTT files in the language of the page are selected by default. They can still select any other available language in the video player.

### Detailed description:

1.  A new `vtt.yml` file is created in the repo, in the `_data/<name of the repo>/` directory.

Example: extract related to [Keyboard Compatibility perspective video](https://www.w3.org/WAI/perspective-videos/keyboard/) :
```yaml
- name: keyboard # Used to identify VTT files for this video, based on the beginning of their basename. Example: keyboard.en.vtt, keyboard_ad_desc.en.vtt, etc.
  captions: # Type of functionality available for this video
    - en #captions' language
  captions-ad:
    - en
  subtitles:
    - fr #languages of the available translations/VTT files.
    - zh-hans
  subtitles-ad:
    - fr
    - zh-hans
  descriptions-ad:
    - en
    - fr
    - zh-hans
```

Note: a symlink to this folder needs to be added in the `wai-website` repository. Example: https://github.com/w3c/wai-website/pull/364/files

When a VTT file is translated in a new language, the translator adds the language shortcode to the list of languages available for the relevant functionality (subtitles, subtitles for the audio-description version, descriptions for the audio-description version, etc.)

2. In the page hosting the video, a different video player include is used (`video-player-data.html` instead of `video-player.html`), with the following new attributes:

- [required] `vtt-data-location`: name of the repo containing the `vtt.yml` file related to this video.
- [required] `vtt-name`: beginning of VTT files basenames
- [optional]  `langfolder`: when present or set to `true`, means VTT files for a language are located in a subfolder named after the language shortcode. Example: `_perspective-videos/cc/en` rather than directly in `_perspective-videos/cc/`

Example:
```liquid
{% include video-player-data.html
    yt-id="7RHG_XiQ0ck"
    yt-id-ad="21yWr7evHTs"
    path="/perspective-videos/cc/"
    lang-folder=true
    vtt-data-location="wai-perspective-videos"
    vtt-name="keyboard"
%}
```

### Opened PRs:

- Example usage in French translation of WAI perspective videos: https://github.com/w3c/wai-perspective-videos/pull/43
- Add new video player include : https://github.com/w3c/wai-website-theme/pull/64
- Full build preview: https://github.com/w3c/wai-website/pull/364
